config/webpack.js

```diff
@@ -0,0 +1,48 @@
+const webpack = require('webpack');
```

Comment:
Is a webpack config needed? Any configs necessary for karma testing could be put in there right? Having a webpack config in each plugin feels like it could get hard to maintain and if we're looking to test actual production environments they should probably be borrowing from the parent webpack config in bpanel since that's what will be doing the build right?

Reply:
The webpack config gets pulled in by karma. I think you may be able to define it inline in the karma config but I don't think that really makes sense, would make the file huge and bloated

Reply:
To run webpack before karma. If they are separate, then you can also run webpack without karma

Reply:
That was basically my question- are we ever running webpack without karma? If not, is webpack necessary?

Reply:
And if we are then yeah, definitely makes sense to keep it.

Reply:
Not a super strong preference, but generally for plugins, webpack/bundling is being done by the parent (bPanel) and so if it's not needed in the project and the formatting can be done by karma, then we can get rid of an unnecessary dependency. If it's necessary for development though then that's a different story. So just wanted to clarify.

Reply:
I'm using:

```
    new webpack.DefinePlugin({
      'process.env.BROWSER': JSON.stringify(true),
    }),
```

To make sure that the proper ledger transport gets used in the `HardwareWallet` class.
To my understanding, you give karma a webpack config and it does webpacking for you when using the karma-webpack module. Do you have any suggestions for other ways of running karma without webpack?

Reply:
gotcha. Well karma doesn't require webpack to run tests but if the tests require webpack bundling then that makes sense to me 👍.